.travis/before_script.sh

```diff
@@ -14,3 +14,20 @@ sudo chown -R travis:travis /var/lib/pulp
 sudo cp .travis/server.yaml /etc/pulp/server.yaml
 
 echo "SECRET_KEY: \"$(cat /dev/urandom | tr -dc 'a-z0-9!@#$%^&*(\-_=+)' | head -c 50)\"" | sudo tee -a /etc/pulp/server.yaml
+
```

Comment:
Why move this before?  If there's a flake8 issue, it will increase the time before the failure is apparent.

Reply:
I wanted to move into before because I wanted to separate the install, migrations and starting services (before), and running of tests into 3 distinct scripts. I then use the install and before scripts in the stage that builds and publishes docs using cron.